### DISCUSSION TITLE
Custom fine tuned embedding models are supported

### DISCUSSION THREAD

**Discussion by kjoth** [michaelfeil/infinity#116] [Q&A] [answered]
Hi,  

Does it support custom fine-tuned embedding models?  Also is there a documentation page available or we have to start the server and get to know via Swagger docs.

**Comment by kjoth** [comment]
Also does infinity support ColBERT ?

**Reply by michaelfeil** [reply, marked as answer]
1. there currently is now documentation beyond the readme available. Just start up the server via docker, the cli is documented 
2. Colbert as late-interaction model is not supported yet.

**Reply by kjoth** [reply]
@michaelfeil Thanks for clarifying. 

Could you clarify does Infinity support custom fine-tuned embedding models?

**Reply by michaelfeil** [reply]
Yes - best if a sbert.net like model on Huggingface…

**Comment by S1LV3RJ1NX** [comment]
@michaelfeil - Do we support colbert now?
